web/modules/custom/dpl_update/dpl_update.deploy.php

```diff
@@ -35,3 +104,53 @@ function dpl_update_deploy_set_branches_not_promoted(): string {
   $count_branches = count($branches_with_empty_promotion_fields);
   return "Set default value for promoted on lists for {$count_branches} branches";
 }
+
+/**
+ * Set default value for all existing eventseries:field_relevant_ticket_manager.
+ */
+function dpl_update_deploy_field_relevant_ticket_manager(): string {
+  $field_name = 'field_relevant_ticket_manager';
+
+  $ids =
+    \Drupal::entityQuery('eventseries')
+      ->accessCheck(FALSE)
+      ->execute();
```

Comment:
I am surprised by the need for this deploy hook. As you have defined a default value for the field I has expected this to apply to all existing entities as well. I guess this is not the case.

Consider if you can refactor this into something like `_dpl_update_set_default_value(string $entity_type, string $field_name, $value)`.

Reply:
A default value is only set on new content.
When you create a new field, all the existing entities still exist, without that field.

That's why you should never assume that a value is set on a field :)

Reply:
I've created it as a helper function now.